src/server/pfs/server/driver.go

```diff
@@ -229,41 +227,34 @@ func (d *driver) inspectRepo(txnCtx *txnenv.TransactionContext, repo *pfs.Repo,
 		return nil, err
 	}
 	if includeAuth {
-		accessLevel, err := d.getAccessLevel(txnCtx.Client, repo)
+		permissions, err := d.getPermissions(txnCtx.Client, repo)
 		if err != nil {
 			if auth.IsErrNotActivated(err) {
 				return result, nil
 			}
 			return nil, errors.Wrapf(grpcutil.ScrubGRPC(err), "error getting access level for \"%s\"", repo.Name)
 		}
-		result.AuthInfo = &pfs.RepoAuthInfo{AccessLevel: accessLevel}
+		result.AuthInfo = &pfs.RepoAuthInfo{Permissions: permissions}
 	}
 	return result, nil
 }
 
-func (d *driver) getAccessLevel(pachClient *client.APIClient, repo *pfs.Repo) (auth.Scope, error) {
+func (d *driver) getPermissions(pachClient *client.APIClient, repo *pfs.Repo) ([]auth.Permission, error) {
 	ctx := pachClient.Ctx()
-	who, err := pachClient.AuthAPIClient.WhoAmI(ctx, &auth.WhoAmIRequest{})
-	if err != nil {
-		return auth.Scope_NONE, err
-	}
-
-	if who.ClusterRoles != nil {
-		for _, s := range who.ClusterRoles.Roles {
-			if s == auth.ClusterRole_SUPER || s == auth.ClusterRole_FS {
-				return auth.Scope_OWNER, nil
-			}
-		}
-	}
 
-	resp, err := pachClient.AuthAPIClient.GetScope(ctx, &auth.GetScopeRequest{Repos: []string{repo.Name}})
+	resp, err := pachClient.AuthAPIClient.Authorize(ctx, &auth.AuthorizeRequest{
```

Comment:
`getPermissions` just tries to `Authorize` with every permission,  and returns the list of permissions that were covered.